src/test/expected/one-assert/expected

```diff
@@ -1 +1 @@
-** 0 of 1 failed (1 iterations)
+** 0 of 2 failed (1 iterations)
```

Comment:
What is going on here? This was supposed to have a single assertion 😆

Reply:
I know... we just can't keep this going, eh? We really need that report filter. This is due to a compiler sanity check. I could just use `any_raw` here to keep the 1 assertion.

Reply:
My bad :( The purpose of this test was to make sure we don't get all the noisy successes for builtin functions, e.g.:
```
<builtin-library-calloc> function calloc
[calloc.assertion.1] assertion false: SUCCESS

<builtin-library-ceil> function ceil
[ceil.assertion.1] assertion false: SUCCESS

<builtin-library-ceilf> function ceilf
[ceilf.assertion.1] assertion false: SUCCESS

<builtin-library-copysign> function copysign
[copysign.assertion.1] assertion false: SUCCESS
...
```
which we got rid of with `--drop-unused-functions`. So to make the test more robust, we need to check if the output does **not** include those lines. Is there a way to do that with our testing infrastructure?

Reply:
At present, there is no way to do that. Not that it would be hard to, but I would wait until we get structured output to implement this kind of checks. The test is still ensuring there are only a couple of successful assertions so I would leave as is.